plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/form/field/Base.scss

```diff
@@ -8,10 +8,25 @@
  * cls: 'x-form-readonly'
  */
 
-.x-form-field[readonly].x-form-field:not([role="combobox"]) {
+@mixin nx-form-field-readonly {
   color: $color-dark-gray;
-  background-color: $color-smoke;
```

Comment:
Most of the predefined roles have read-only fields.

I think there’s a difference between an immutable field and a field which you don’t have permissions to edit (or can’t due to some other condition). I’m okay with immutable fields getting the nx-no-border treatment (since they are, essentially, labels), but I think all other readonly fields need to look like input boxes.

If a readonly input field doesn’t have a grey background, it looks the same as an empty field with placeholder text, which is the reason I added a smoke background.

![roles - sonatype nexus 2015-05-08 17-11-37](https://cloud.githubusercontent.com/assets/186715/7540376/78f97320-f5a5-11e4-97fa-7750f51c2597.png)